### PR TITLE
Added API to fetch all task and action nodes

### PR DESCRIPTION
**Pull request by alwynmathew**

### CODE REVIEW

File: dtp_apis/fetch_DTP_API.py [hunk 1]
```diff
@@ -583,6 +583,60 @@ def fetch_activity_nodes(self, url=None):
...
+        ------
+        dictionary
+            JSON mapped to a dictionary. The data contain task nodes.
+        """
+
+        payload = json.dumps({
+            "query": {
+                "$domain": self.DTP_CONFIG.get_domain(),
+                "$classes": {
+                    "$contains": self.DTP_CONFIG.get_ontology_uri('task')
```
Comment (jschlenger) [inline dtp_apis/fetch_DTP_API.py:605]: does this work with lowercase here?
The URL is "https://www.bim2twin.eu/ontology/Core#Task"
I don't know if the lowercase will be converted correctly to uppercase.

File: dtp_apis/fetch_DTP_API.py [hunk 2]
```diff
@@ -583,6 +583,60 @@ def fetch_activity_nodes(self, url=None):
...
+        ------
+        dictionary
+            JSON mapped to a dictionary. The data contain action nodes.
+        """
+
+        payload = json.dumps({
+            "query": {
+                "$domain": self.DTP_CONFIG.get_domain(),
+                "$classes": {
+                    "$contains": self.DTP_CONFIG.get_ontology_uri('asPerformedAction')
```
Comment (jschlenger) [inline dtp_apis/fetch_DTP_API.py:632]: the class name is just "Action" and not asPerformedAction

**Review by jschlenger — CHANGES_REQUESTED** [review]
Two small change requests

File: dtp_apis/fetch_DTP_API.py (hunk 1 above)
Comment (alwynmathew) [reply dtp_apis/fetch_DTP_API.py:605]: `task` is a string mapped to task IRI on config `<URI function="task" type="xs:anyURI">https://www.bim2twin.eu/ontology/Core#Task</URI>`

File: dtp_apis/fetch_DTP_API.py (hunk 2 above)
Comment (alwynmathew) [reply dtp_apis/fetch_DTP_API.py:632]: `asPerformedAction ` is a string mapped to task IRI on config `<URI function="asPerformedAction" type="xs:anyURI">https://www.bim2twin.eu/ontology/Core#Action</URI>`. This is how @copyme defined URIs in the config file.

**Pull request merged**